src/runtime_src/driver/xclng/drm/xocl/subdev/mailbox.c

```diff
@@ -1328,6 +1408,12 @@ int mailbox_request(struct platform_device *pdev, void *req, size_t reqlen,
 		*resplen = respmsg->mbm_len;
 
 	free_msg(respmsg);
+
+	/* Sleep so the next request doesn't come before the response.
+	 * 10 ms was too little, 20 ms works.
+	 */
+	msleep_interruptible(20);
```

Comment:
this is only for confirming the issue, not something you can use in production code!!